### DISCUSSION TITLE
Question about License

### DISCUSSION THREAD

**Discussion by sanunes27** [mifi/lossless-cut#2169] [General]
Hello! This is an amazing application for my use case, but I have a question about the License. I understand it is free when downloading via GitHub, but is there any other requirement such as making attribution or something? Or is it really free that I could even use it commercially? I run a YouTube channel, and this app will allow me to merge multiple videos into one, which will save me a lot of time.

Thanks!

**Comment by mifi** [comment]
Hi. It is free and open source so you can use it for pretty much what you want as long as you follow its GPL license. Merging files for youtube is perfectly fine

**Reply by Jay123210599** [reply]
Can you answer this question for me?
https://github.com/mifi/lossless-cut/discussions/2156
